google_drive/public/js/google_docs.js

```diff
@@ -26,12 +29,20 @@ function GoogleDocumentBlock(runtime, element) {
             url: runtime.handlerUrl(element, 'publish_event'),
             data: JSON.stringify({
                 url: document_url,
-                displayedin: presented_within,
-                event_type: 'edx.googlecomponent.document.displayed',
-             })
+                displayed_in: presented_within,
+                event_type: 'edx.googlecomponent.document.displayed'
+            }),
+            success: function(){
+                ajaxflag.val("1");
+            }
         });
     }
-    iframe.load(function(e){SignalDocumentLoaded(e, 'iframe');});
-    image.load(function(e){SignalDocumentLoaded(e, 'img');});
+
+    $('iframe', element).load(function(e){
```

Comment:
I made this change while I was writing tests, but obviously forgot to change it back. I will revert this change.